README.md

```diff
@@ -20,18 +20,18 @@ Follow these steps to run Reconciler locally:
 2. Run Reconciler with the default configuration. By default, Reconciler installs all components listed in the [`components.yaml`](https://github.com/kyma-project/kyma/blob/main/installation/resources/components.yaml) file.
 
    ```bash
-   ./bin/reconciler-darwin local 
+   ./bin/mothership-darwin local 
    ```
 
    > **CAUTION:** At the moment, reconciliation with default values will fail. You must specify additional parameters:
    > ```bash
-   > ./bin/reconciler-darwin local global.ingress.domainName=example.com,global.domainName=example.com,tracing.authProxy.config.useDex=false,tracing.virtualservice.enabled=false,kiali.authProxy.config.useDex=false,kiali.virtualservice.enabled=false
+   > ./bin/mothership-darwin local global.ingress.domainName=example.com,global.domainName=example.com,tracing.authProxy.config.useDex=false,tracing.virtualservice.enabled=false,kiali.authProxy.config.useDex=false,kiali.virtualservice.enabled=false
    > ```
 
 
    Alternatively, you can run Reconciler with the specified components:
```

Comment:
I suggest to add indentation to this paragraph so that the alternative is also part of the Caution note.